tekton/publish.yaml

```diff
@@ -177,6 +183,8 @@ spec:
         IMAGE_WITHOUT_SHA_AND_TAG=${IMAGE_WITHOUT_SHA%%:*}
         IMAGE_WITH_SHA=${IMAGE_WITHOUT_SHA_AND_TAG}@${IMAGE##*@}
 
+        echo $IMAGE_WITH_SHA, >> $(results.IMAGES.path)
+
         if [[ "$(params.releaseAsLatest)" == "true" ]]
         then
           crane cp ${IMAGE_WITH_SHA} ${IMAGE_WITHOUT_SHA_AND_TAG}:latest
```

Comment:
Comparing this to Chains publish task, we are missing REGION specific sha in the result here

https://github.com/tektoncd/chains/blob/main/release/publish.yaml#L184

Do we need to add one more `echo` command here?

Reply:
I'd recommend adding that in so that all images are signed

Reply:
Amazing, thanks again - I didn't copy verbatim enough 😆 

Added the additional region `echo`